migration/iD10/oracle/202209090852_IDEMPIERE-5410.sql

```diff
@@ -0,0 +1,410 @@
+;
+
+-- Sep 9, 2022, 8:52:44 AM CEST
+INSERT INTO AD_ChartDatasource (AD_Client_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,DateColumn,EntityType,IsActive,KeyColumn,Name,AD_ChartDatasource_ID,FromClause,Updated,UpdatedBy,WhereClause,AD_Chart_ID,ValueColumn,TimeOffset,AD_ChartDatasource_UU) VALUES (0,0,259,TO_TIMESTAMP('2022-09-09 08:52:44','YYYY-MM-DD HH24:MI:SS'),100,'o.DateOrdered','D','Y','o.C_Order_ID','Current',200001,'C_Order o',TO_TIMESTAMP('2022-09-09 08:52:44','YYYY-MM-DD HH24:MI:SS'),100,'o.DocStatus IN (''CO'',''CL'') AND o.IsSOTrx=''Y''',200001,'count(o.C_Order_ID)',0,'6a2ed1ff-2ba7-4e2d-ac53-97e6b9715e0d')
+;
+
+-- Sep 9, 2022, 8:52:45 AM CEST
+INSERT INTO AD_Chart (AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Chart_ID,ChartType,Name,WinHeight,IsDisplayLegend,ChartOrientation,IsTimeSeries,TimeUnit,TimeScope,DomainLabel,RangeLabel,AD_Chart_UU,EntityType) VALUES (0,0,'Y',TO_TIMESTAMP('2022-09-09 08:52:44','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2022-09-09 08:52:44','YYYY-MM-DD HH24:MI:SS'),100,200002,'WC','User Sessions',300,'Y','H','Y','M',0,'Days','Sessions','3edf7ef4-04b7-4356-8345-08122aa1a693','D')
+;
+
+-- Sep 9, 2022, 8:52:45 AM CEST
+INSERT INTO AD_ChartDatasource (AD_Client_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,DateColumn,EntityType,IsActive,KeyColumn,Name,AD_ChartDatasource_ID,FromClause,Updated,UpdatedBy,AD_Chart_ID,ValueColumn,TimeOffset,SeriesColumn,AD_ChartDatasource_UU) VALUES (0,0,566,TO_TIMESTAMP('2022-09-09 08:52:45','YYYY-MM-DD HH24:MI:SS'),100,'s.created','D','Y','s.AD_Session_ID','Session',200002,'AD_Session s
+INNER JOIN AD_User u ON (u.AD_User_ID=s.createdby)',TO_TIMESTAMP('2022-09-09 08:52:45','YYYY-MM-DD HH24:MI:SS'),100,200002,'count(1)',0,'u.name','1ee4a54c-04b7-4668-a3d6-d85c77652738')
+;
+
+-- Sep 9, 2022, 8:52:46 AM CEST
+INSERT INTO AD_Chart (AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Chart_ID,ChartType,Name,WinHeight,IsDisplayLegend,ChartOrientation,IsTimeSeries,TimeUnit,TimeScope,DomainLabel,RangeLabel,AD_Chart_UU,EntityType) VALUES (0,0,'Y',TO_TIMESTAMP('2022-09-09 08:52:45','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2022-09-09 08:52:45','YYYY-MM-DD HH24:MI:SS'),100,200003,'AC','Orders last 30 days',300,'Y','V','Y','W',0,'Day','Value','f046ce9b-7cef-4085-b479-e47c690f2585','D')
+;
+
+-- Sep 9, 2022, 8:52:46 AM CEST
+INSERT INTO AD_ChartDatasource (AD_Client_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,DateColumn,EntityType,IsActive,KeyColumn,Name,AD_ChartDatasource_ID,FromClause,Updated,UpdatedBy,AD_Chart_ID,ValueColumn,TimeOffset,AD_ChartDatasource_UU) VALUES (0,0,259,TO_TIMESTAMP('2022-09-09 08:52:46','YYYY-MM-DD HH24:MI:SS'),100,'o.dateordered','D','Y','o.C_Order_ID','current',200003,'C_Order o',TO_TIMESTAMP('2022-09-09 08:52:46','YYYY-MM-DD HH24:MI:SS'),100,200003,'sum(o.grandtotal)',0,'7f05953d-8591-42d2-b6a3-a27a3381f797')
+;
+
+-- Sep 9, 2022, 8:52:47 AM CEST
+DELETE FROM AD_Chart WHERE AD_Chart_ID=200003
```

Comment:
this line is a mismatch vs the postgresql version

Note that normally I use these scripts to check problems with the migration scripts:
https://github.com/globalqss/idempiere-stuff/tree/stuff/verifyPullRequest